app/templates/packages/view.html

```diff
@@ -264,7 +297,7 @@ <h1 class="display-3">
 							{% for ss in screenshots %}
 								{% if ss.approved or package.check_perm(current_user, "ADD_SCREENSHOTS") %}
 									<li>
-										<a href="{{ ss.url }}" class="gallery-image">
+										<a href="#" data-bs-toggle="modal" data-bs-target="#galleryModal" data-bs-slide-to="{{ loop.index - 1 }}" class="gallery-image">
```

Comment:
ContentDB follows [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement) - these gallery images should continue to work the same as before this PR when JS is disabled. 

To prevent opening the link, in the JS you can either change the href attribute or call e.preventDefault();   (I think, needs testing)

```suggestion
										<a href="{{ ss.url }}" data-bs-toggle="modal" data-bs-target="#galleryModal" data-bs-slide-to="{{ loop.index - 1 }}" class="gallery-image">
```